### ISSUE TITLE
letter э

### ISSUE THREAD

**Issue by greengrocer98** [area/language-latex#191]
### Prerequisites
- [X ] Put an `X` in this box if you have done the following:
  - Ensured the issue has not already been raised
  - Ensured the issue is caused by this package

### Description
Russian letter э replaces by the symbols `' and I don't know how to fix this problem. Firstly, I thought it may cause of atom encoding, but in plain text mode letter э works correctly

### Steps to reproduce & Minimum working example
1.

![image](https://user-images.githubusercontent.com/36965345/45595350-df82ab00-b9b2-11e8-84e1-bb4a1e57263e.png)

![image](https://user-images.githubusercontent.com/36965345/45595364-fd501000-b9b2-11e8-9cd2-bd7e3245675e.png)

#### Expected behaviour
э

#### Actual behaviour
`'

**Comment by Aerijo** [comment]
@greengrocer98 Hmm. I could not reproduce the issue using the Russian keyboard. Have you tried disabling all other packages?

**Comment by greengrocer98** [comment]
@Aerijo yeah, I've tried to disable latextools just to write the code and also tried to disable latex-language. In the second case letter э works correctly. I've just noticed the same problem in language-latexsimple. Maybe it can cause of Unicode encoding of symbol quote and letter э, because they have the same place on the keyboard?

**Comment by AntipovDen** [comment]
Previously I had this issue as well, but after a long period of not writing in Russian I cannot reproduce it. As far as I remember, it was somehow connected with the mechanism of autocomplete of the paired symbols (not sure that it is the right name for it, but it is when you write an open bracket, atom automatically writes a closing bracket after the cursor). So the latex package redefined some of such autocompletions, like for the dollar symbol and there was the same for the apostrophe symbol, but the first apostrophe was also substituted with a grave accent symbol (`). And the thing is that 'э' is on the same button as the apostrophe. So it seems like that mechanism was just trying to read the code of the pressed button without respect to the layout (not sure, how it really works, sorry).

There is also a chance that back then I might use some other latex package in parallel (I know, it is not a clever decision), so this issue may be not related to this particular package. At least I cannot find this feature in the preferences.
